src/main.rs

```diff
@@ -83,33 +77,66 @@ fn inner_main() -> Result<(), Error> {
         let rgb = Rgb::from_hsv(new_colour); // Turn HSV into RGB.
 
         // Make f64s into i64s.
-        let r = (rgb.red * 100.0) as i64;
-        let g = (rgb.green * 100.0) as i64;
-        let b = (rgb.blue * 100.0) as i64;
+        let r = rgb.red * 100.0;
+        let g = rgb.green * 100.0;
+        let b = rgb.blue * 100.0;
 
         // If any value would drop below zero (which we can't display), continue.
-        if r < 0 || g < 0 || b < 0 {
+        if r < 0.0 || g < 0.0 || b < 0.0 {
             continue;
         }
 
-        #[cfg(not(windows))] { // Print on all systems, except for Windows.
+        let (r, g, b) = (r as u8, g as u8, b as u8);
+
+        #[cfg(not(gtk))] { // Print on all systems, except for Windows.
             println!(
-                "\x1b[48;2;{r};{g};{b}m   #{r:02X}{g:02X}{b:02X}   \x1b[0;0m",
+                "\x1b[38;2;{ri};{gi};{bi}m\x1b[48;2;{r};{g};{b}m   #{r:02X}{g:02X}{b:02X}   \x1b[0;0m",
                 r = r,
                 g = g,
-                b = b
+                b = b,
+                ri = 255 - r,
+                gi = 255 - g,
+                bi = 255 - b
             );
         }
-        #[cfg(windows)] { // Make GTK Labels on Windows.
-            let label = Label::new("hello");
+        #[cfg(gtk)] { // Make GTK Labels on Windows.
+            use std::fmt::Write;
+
+            let mut string = String::with_capacity(1 + 2*3);
+            write!(string, "#{:02X}{:02X}{:02X}", r, g, b).unwrap();
+
+            let label = Label::new(&*string);
+            label.override_color(StateFlags::NORMAL, &RGBA {
+                red: 1.0 - rgb.red,
+                green: 1.0 - rgb.green,
+                blue: 1.0 - rgb.blue,
+                alpha: 1.0
+            });
+            label.connect_draw(move |label, ctx| {
+                let rect = label.get_allocation();
+                println!("{:?}", rect);
+                println!("{:?}", rgb);
+
+                ctx.rectangle(0.0, 0.0, rect.width as f64, rect.height as f64);
+                ctx.set_source_rgb(rgb.red, rgb.green, rgb.blue);
+                ctx.fill();
+
+                ctx.set_source_rgb(1.0 - rgb.red, 1.0 - rgb.green, 1.0 - rgb.blue);
+                ctx.show_text("YEE");
```

Comment:
Although this "YEE"?

Reply:
Whoops. Well, the secret is out. My debug messages are the weirdest.

Reply:
kek!

Reply:
Yes the rest of my debug messages are `kek`. And `lol`, `wat`, `wot`, `woot`, `hey`, `hi`, `1`, `2`, et.c.